Corona-Warn-App/src/deviceForTesters/java/de/rki/coronawarnapp/test/contactdiary/ui/ContactDiaryTestFragment.kt

```diff
@@ -42,6 +47,29 @@ class ContactDiaryTestFragment : Fragment(R.layout.fragment_test_contact_diary),
             normalPersonEncountersButton.setOnClickListener { vm.createPersonEncounters(false) }
             locationVisitsCleanButton.setOnClickListener { vm.clearLocationVisits() }
             personEncountersCleanButton.setOnClickListener { vm.clearPersonEncounters() }
+            durationValue.setOnClickListener {
+                val args = Bundle()
+                args.putString(ContactDiaryDurationPickerFragment.DURATION_ARGUMENT_KEY,
+                    binding.durationValue.text.toString())
+
+                val durationPicker = ContactDiaryDurationPickerFragment()
+                durationPicker.arguments = args
+                durationPicker.setTargetFragment(this@ContactDiaryTestFragment, 0)
```

Comment:
We should not use this, could you please use `navbackstackentry`
https://developer.android.com/guide/navigation/navigation-programmatic#navbackstackentry